sechub-administration/src/test/java/com/daimler/sechub/domain/administration/project/ProjectAdministrationRestControllerMockTest.java

```diff
@@ -162,6 +178,38 @@ public void delete_project_calls_delete_service() throws Exception {
         verify(projectDeleteService).deleteProject("projectId1");
     }
 
+    @Test
+    public void change_project_calls_change_details() throws Exception {
+
+        /* execute + test @formatter:off */
+        
+        this.mockMvc.perform(
+                put(https(PORT_USED).buildAdminChangesProjectDescriptionUrl(PROJECT_ID.pathElement()), "project1").
```

Comment:
Why are you building a project description URL? The method signature indicates to me you are changing the project details. It seems to me that the method is named incorrectly.